### ISSUE TITLE
GRID Autosport Xbox 360 NeFS File Encryption

### ISSUE THREAD

**Issue by umustbeloggedintododat** [victorbush/ego.nefsedit#27]
Trying to open the NeFS files of GRID Autosport Xbox 360 DLC results in an error, could it be possible new encryption?
I also tried the QuickBMS NeFS extractor, just says no files extracted.
Console log:
`[19:47:53 INF] ----------------------------
[19:47:53 INF] Saving settings...
[19:47:53 INF] Settings saved.
[19:48:11 INF] ----------------------------
[19:48:11 INF] Saving settings...
[19:48:11 INF] Settings saved.
[19:48:11 INF] ----------------------------
[19:48:11 INF] Opening archive: D:\Grid.Autosport.DLC.RF.X360-ZTM\434D0851\00000002\Road & Track Car Pack\cam_high.nefs
[19:48:11 INF] Header magic number mismatch, assuming header is encrypted.
[19:48:11 ERR] Failed to open archive D:\Grid.Autosport.DLC.RF.X360-ZTM\434D0851\00000002\Road & Track Car Pack\cam_high.nefs.
Could not find any recognizable digits.
`
Sample file (Rename to NeFS because github): [cam_high.txt](https://github.com/victorbush/ego.nefsedit/files/11658628/cam_high.txt)

**Comment by TheNFSModder217132** [comment]
no, the dlc nefs use the exact same encyption as pc (to be clear: RSA (for intro) and AES-256 (for the rest of the header)

however it's because nefsedit doesn't support big endian archives (which is what 360 and PS3 use due to architecture (PPC)) (still doesn't for NeFS 1.6) (that is afaik, but for aes, definitely)

**Comment by ptasev** [comment]
I'm slowly making progress on 1.6 big endian.
